sandpack-react/rollup.config.js

```diff
@@ -25,11 +25,14 @@ const configBase = {
 
   plugins: [
     typescript({ tsconfig: "./tsconfig.json" }),
-    replace({ "process.env.TEST_ENV": "false" }),
+    replace({
+      preventAssignment: true,
```

Comment:
Why this change is relevant for this PR?

Reply:
i migrated `rollup-plugin-replace` to `@rollup/plugin-replace` in monorepo

`preventAssignment` flag is new in this plugin, it doesn't replace left values in the assignment to prevent bugs.